tests/node/util/BsonTest.js

```diff
@@ -0,0 +1,12 @@
+const SDL = require('./../../../lib/js/dist/SDL.js');
+const Bson = SDL.util.Bson;
+const expect = require('chai').expect;
+
+
+describe('BsonTest', function () {
+    it('serialize deserialize test', function (done) {
```

Comment:
Test name does not match naming convention used in other tests. e.g. `testThingMethodBehavior`

Reply:
testSerialize and testDeserialize used as test names now.